### PR TITLE
Add faq page for Experience

### PR DESCRIPTION
**Pull request by amir-hadzic**
Here's the preview:

localhost_8000_experience_faq_

### CODE REVIEW

File: docs/experience/faq.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+No, the Experience script will not negatively impact your website performance.
+We will do our best to ensure that the script is as minimal as possible and
+that it never interferes with the normal operation of your website or webapp.
+The script is loaded asynchronously and in most cases should have no or very
+small effect on website performance.
+
+<a name="multiple-domain" href="multiple-domain" class="faq-questions"><strong>Can I send data from a different domain?</strong></a>
+
+Depending on your plan, you may be able to send data from multiple domains at
+the same time. If your pricing plan supports more than one domain, just setup
```
Comment (adnanrahic) [inline docs/experience/faq.md]: setup -> set up

File: docs/experience/faq.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+the script on additional domains and your data should appear shortly in
+Experience.
+
+If you want to switch the domain from which you are sending data, note that it
+may take up to 5 minutes for the data to start being accepted if your pricing
+plan supports only one domain.
+
+<a name="configure-requirements" href="configure-requirements" class="faq-questions"><strong>What are the user satisfaction scores shown in Experience based on?</strong></a>
+
+The scores are based on the time threshold defined in [Configure
```
Comment (adnanrahic) [inline docs/experience/faq.md]: in -> in the

File: docs/experience/faq.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+[Configure Requirements](/experience/configure-requirements) to learn how to
+adjust the time thresholds or [User
+Satisfaction](/experience/user-satisfaction) to learn more about Apdex scores
+in general.
+
+### Troubleshooting
+
+<a name="stopped-sending-data" href="#stopped-sending-data"><strong>My App has stopped receiving data. What should I do?</strong></a>
+
+Check if you have reached your daily limit in App settings -> Usage screen. If
```
Comment (adnanrahic) [inline docs/experience/faq.md]: in -> in the

**Comment by amir-hadzic** [comment]
Thanks @adnanrahic

**Pull request merged**